index.js

```diff
@@ -45,6 +45,7 @@ const defaultLookups = {
  * @param {String} [options.webpackConfig] Path to the webpack config
  * @param {Object} [options.ast] A preparsed AST for the file identified by filename.
  * @param {Object} [options.tsconfig] Path to a typescript config file
+ * @param {Object} [options.fileSystem] An alternative filesystem / fs implementation to use for locating files.
```

Comment:
In precinct, we called this option `fs`. Should we name this the same for consistency? It would also make it easier to set the option across both cabinet and precinct.

Reply:
We'd also need this documented in the readme.

Reply:
Doh! In other dependencies I have already had `fileSystem` as an option merged. In face I thought I was being consistent by using `fileSystem` as the option property name.. I think there will be some mix and match now in a few places between `fs` and `fileSystem`. I'll change this to `fs` to be consistent with precinct.

Reply:
I'll make the change in precinct to be `fileSystem` and keep `options.fs` undocumented for back compat.